main.go

```diff
@@ -60,6 +61,11 @@ func initLogger(logLevel string) error {
 	return nil
 }
 
+func init() {
+	// Force coredump + full stacktrace on internal error
+	debug.SetTraceback("crash")
```

Comment:
Does it show up through stderr?

Reply:
That's a good question. afaics, when an internal error occurs, the go runtime calls [`builtin.print()`](https://golang.org/pkg/builtin/#print), which does indeed write to stderr:

- https://github.com/golang/go/blob/master/src/runtime/signal_sighandler.go#L89..L135

Reply:
@jodh-intel I assume that if we set a logger with logrus, those traces will go to this defined logger right ?
Otherwise we'll get the error printed on stderr of our console, instead of the journal, which is not the expected behavior.

Reply:
I don't believe so. This is a golang facility which doesn't provide us with a lot of control. The ideal would obviously be to log the error but as this is an unrecoverable failure scenario, the best we can do (easily) I think is squirt the data back to the user. We *could* start playing games like modifying the value of `Stderr` I guess.

Reply:
Okay I get it. Then this is fine for now, but we could investigate how to make this being redirected through the logrus logs.

Reply:
Can we override `stderr` with a real local file or something like syslogd in such case? Sending backtrace to user's stderr might leak security bugs. If not, IMO we should make the dump configurable and do not enable it in a production environment.

Reply:
@bergwolf - this fix is too simplistic. I'm closing this and replacing it with #59.